Makefile

```diff
@@ -55,7 +52,7 @@ test:
 	go test -v ./pkg/...
 
 .PHONY: ensure-gopath
-check-gopath:
+ensure-gopath:
 ifndef GOPATH
-    $(error GOPATH must be set)
+	$(error GOPATH must be set)
```

Comment:
Why the double tabs?

Reply:
Not double, switched spaces to a tab as per Makefile convention